docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -329,12 +329,25 @@ function my_plugin_block_categories( $categories, $post ) {
 			array(
 				'slug' => 'my-category',
 				'title' => __( 'My category', 'my-plugin' ),
-				'icon'  => '<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="none" d="M0 0h24v24H0V0z" /><path d="M19 13H5v-2h14v2z" /></svg>',
+				'icon'  => 'wordpress',
 			),
 		)
 	);
 }
 add_filter( 'block_categories', 'my_plugin_block_categories', 10, 2 );
 ```
 
-You can also display an icon with your block category by setting an `icon` attribute. The value can be the slug of a [WordPress Dashicon](https://developer.wordpress.org/resource/dashicons/), or a custom `svg` element.
+You can also display an icon with your block category by setting an `icon` attribute. The value can be the slug of a [WordPress Dashicon](https://developer.wordpress.org/resource/dashicons/).
+It is possible to set an SVG as the icon of the category if a custom icon is needed.
+To do so, the icon should be rendered and set on the frontend, so it can make use of WordPress SVG, allowing mobile compatibility and making the icon more accessible.
+To use the SVG icon with the category as shown in the previous example, add the following example JavaScript code to the editor calling `wp.blocks.updateCategory` e.g:
```

Comment:
Would be good to have line breaks between paragraphs, or put them all as one line if they should be the same paragraph.

Reply:
Thank you @chrisvanpatten the comments were updated 👍